### ISSUE TITLE
Endpoints of fuel lines wrong

### ISSUE THREAD

**Issue by dertroglodyt** [UbioWeldingLtd/UbioWeldContinued#29]
Fuel lines seem to work ok. 
Just the endpoints are positioned wrong.

**Comment by Alewx** [comment]
fuel lines are not needed in weldments and should be removed completely.

**Comment by dertroglodyt** [comment]
Shure they "work", its one part after all (facepalm).
Just love the optics of them. Would be nice to have them even if not needed.

**Comment by Alewx** [comment]
if you have an idea for the vectored direction for them that is possible.

**Comment by dertroglodyt** [comment]
Sorry, would like to help but I have no insight into the graphics part of unity/ksp whatsoever.

**Comment by Alewx** [comment]
fuel lines and struts are quite easy done,

they work that way that they have the part at a position with a vector3 and then there is vector3 that defines the direction that it will go until it hits any other part.

**Comment by dertroglodyt** [comment]
Then my guess would be that the second vec3 isn't stored / gets mixed up. Because the starting points of fuel lines are correct.

**Comment by Alewx** [comment]
yep but any code suggestion for getting the direction vector right? ;)

**Comment by dertroglodyt** [comment]
Sorry, although I'm a programmer I have no experience with c# and have too little time for delving into both c# and unity. Java and lwjgl would be tempting since I already did some work there in my free time.

**Comment by Alewx** [comment]
c# ist just like java but better structured:)
